### PR TITLE
Add unit test for record.go

### LINKED ISSUE #1198: add unit tests for record.go
/kind feature
/area provider/ibmcloud

**Describe the solution you'd like**
I would like to add unit tests for `pkg/record/record.go`

**Anything else you would like to add:**
NA

### PR DESCRIPTION
**Pull request by Anddd7**
**What this PR does / why we need it**:

Add unit test for `pkg/record/record.go`.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1198 

**Special notes for your reviewer**:

/area provider/ibmcloud

### CODE REVIEW

File: pkg/record/record_test.go [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+package record
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	runtime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	cgrecord "k8s.io/client-go/tools/record"
```
Comment (Karthik-K-N) [inline pkg/record/record_test.go:25]:
```suggestion
	"testing"

	"github.com/stretchr/testify/require"

	"k8s.io/apimachinery/pkg/runtime"
	"k8s.io/apimachinery/pkg/runtime/schema"
	cgrecord "k8s.io/client-go/tools/record"
```

**Comment by Karthik-K-N** [comment]
@Anddd7 Thank you for submitting a PR to add unit testcases.

File: pkg/record/record_test.go (hunk 1 above)
Comment (Karthik-K-N) [reply pkg/record/record_test.go:25]: Lets reorder the import to be in sych with other files.

**Review by Karthik-K-N — COMMENTED** [review]
Other than this LGTM

**Comment by mkumatag** [comment]
Also please fix the commit message - remove the fix and PR number

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**